### DISCUSSION TITLE
Movistar LaLiga

### DISCUSSION THREAD

**Discussion by R4IN2007** [iptv-org/community#343] [Show and tell]
### Channel Name

Movistar LaLiga

### Channel ID

LaLigaporMovistarPlus.es

### Country

Spain

### Language

Spanish

### Website

https://www.movistar.es/

### Stream URL

http://93.152.174.144:4000/play/beinlaliga/index.m3u8

**Comment by what-a** [comment]
Same channel or different?
![image](https://user-images.githubusercontent.com/88639906/151908666-9c5d5564-1ab7-4917-a923-9dee91763185.png)

**Comment by R4IN2007** [comment]
Different Channel

**Comment by what-a** [comment]
> Different Channel

Thank you for confirming.
BTW, the proposed stream will not work for me here in California. It might be Geo-blocked.

**Comment by BellezaEmporium** [comment]
Astra server channel, works on my side.

Playlist : http://93.152.174.144:4000/playlist.m3u8 Pretty much a mix of European channels up there (i can see BBC UK, Pinko IT, beIN UK, ES, FR and some others)

**Comment by AntiPontifex** [comment]
Dead issue: please close.

**Comment by freearhey** [comment]
http://93.152.174.144:4000/play/beinlaliga/index.m3u8 (offline)
<img width="393" alt="image" src="https://user-images.githubusercontent.com/7253922/232223905-4aff83dc-d8ae-4c7b-9aed-944669adcbfb.png">

**Comment by vicentenaldatorrijo-debug** [comment]
Hello, someone have the new
